docs/io/votable/dataorigin.rst

```diff
@@ -0,0 +1,96 @@
+.. _astropy-io-votable-dataorigin:
```

Comment:
If I have to guess, your chose ref here conflicts with API tag (not 100% sure). Try renaming this and see. 🤞 
```suggestion
.. _astropy_io_votable_dataorigin_intro:
```